examples/configs/grpo_math_1B.yaml

```diff
@@ -106,6 +106,7 @@ policy:
       pipeline_parallel_size: 1
       gpu_memory_utilization: 0.6
       max_model_len: ${policy.max_total_sequence_length}
+      enforce_eager: True
```

Comment:
@jiemingz why are we enforcing eager if we want cuda graphs to be enabled by default?